CONTRIBUTING.md

```diff
@@ -61,6 +61,8 @@ Pull requests should not be squash merged in order to keep complete commit messa
 This means that, when addressing review comments in order to keep the pull request in an always mergeable status, you will sometimes need to rewrite your branch's history and then force-push it with `git push --force-with-lease`.
 Useful git commands that can help a lot with this are `git commit --patch --amend` and `git rebase --interactive`. For more details consult the git man pages or online resources like [git-rebase.io](https://git-rebase.io/) or [The Pro Git Book](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History).
 
+If you add yourself as maintainer, use `maintainers: add <handle>` as commit message ([see also](https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers).
```

Comment:
Yes, I would put the commit message format in as an example and change the template accordingly under "Submitting Changes" and also try to make it more obvious that changes to `maintainer-list.nix` should occur in a separate commit.

I think it might be reasonable to also normalize `doc/<page>:` and `lib/<module>:` commit messages or simply `doc:` and `lib:` for more general changes, but that isn't necessary here.

Reply:
Ah yeah then I guess this fits very well into the scope of the contributing documentation cleanup in #245243, because I create separate contributing documents for each part of Nixpkgs, such as `pkgs/README.md`, `doc/README.md`, `maintainers/README.md`, etc. And this would fit very well into `maintainers/README.md`.

Reply:
I added the change requested by @infinisil as a commit to this PR, which I can squash if requested of course.